portal/src/hook/usePivot.ts

```diff
@@ -14,11 +14,13 @@ export function usePivot(): {
     (item) => {
       const itemKey = item?.props.itemKey;
       if (typeof itemKey === "string") {
-        navigate(`./#${itemKey}`);
+        // NOTE: escape fragment for safety
+        // eslint-disable-next-line  no-useless-escape
+        navigate(`./\#${itemKey}`);
```

Comment:
I mean escape the fragment using percent encoding: https://stackoverflow.com/questions/13861273/how-should-fragment-part-of-a-url-be-escaped

Reply:
I see, I misunderstood your previous comment, updated to using RFC3986